### PR TITLE
Add graphical empty state into block inserter

### PR DESCRIPTION
**Pull request by truchot**
## Description

Add graphical empty state into inserter block related to #17122 

## Screenshots
localhost_1234_

## Types of changes
I added `::before` pseudo-element to create graphical empty state and I centered vertically and horizontally the "no results" block.

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation.
- [ ] I've included developer documentation if appropriate.

### CODE REVIEW

**Comment by mapk** [comment]
Thanks for taking this on, @truchot! First, the "no blocks" state looks fantastic.

Screen Shot 2019-09-05 at 7 44 35 AM

However, the other states of the Inserter/Library are now centering awkwardly.

Screen Shot 2019-09-05 at 7 43 55 AM
 
And, when searching a finding a block:

Screen Shot 2019-09-05 at 7 44 25 AM

It looks to me like the centering you applied might be bleeding into the rest of the Inserter?

File: packages/block-editor/src/components/inserter/style.scss [hunk 1]
```diff
@@ -89,6 +89,9 @@ $block-inserter-search-height: 38px;
 }
 
 .block-editor-inserter__results {
+	align-items: center;
+	justify-content: center;
+	display: flex;
```
Comment (mapk) [inline packages/block-editor/src/components/inserter/style.scss]: This looks like where the centering problem exists. It's effecting all items in the Inserter.

Comment (truchot) [reply packages/block-editor/src/components/inserter/style.scss]: Yes I fix it in the next commit :) Thanks !

**Comment by kjellr** [comment]
Thanks, @truchot! In general, this looks great. I'm seeing just one last bug: double lines in between panels.

**This PR** 
 
Screen Shot 2019-09-05 at 3 46 28 PM

**Master**

Screen Shot 2019-09-05 at 3 46 58 PM

**Comment by truchot** [comment]
I fix it :)

**Review by kjellr — COMMENTED** [review]
Sorry for the delayed review, @truchot! I've added a couple comments, but have a more general implementation note, too:

After thinking about this more, I think it makes more sense to use an SVG instead of drawing it this way. Doing so would ensure that this icon would align to any other Material-based question mark icons we use elsewhere in the UI. 

It would be best to do this in JS, but alternatively if we want to stick to CSS, we could we try inlining [the SVG](https://cloudup.com/iwXO7g6QAjb) instead. Something like: 

```
content: url('data:image/svg+xml;utf8,<svg width='24' height='24' viewbox='0 0 24 24' xmlns='http://www.w3.org/2000/svg'><path d='M8 10c0-2.21 1.79-4 4-4s4 1.79 4 4c0 1.283-.79 1.973-1.56 2.646C13.712 13.283 13 13.905 13 15h-2c0-1.821.942-2.543 1.77-3.178.65-.498 1.23-.943 1.23-1.822 0-1.1-.9-2-2-2s-2 .9-2 2H8zM13 16v2h-2v-2h2z' fill='#6c7781'/></svg>);
```

**Comment by truchot** [comment]
Thanks @kjellr ! I fix spacing with grid size variables and I display icon within JS as you mentioned.

**Comment by Soean** [comment]
@truchot Oh, no feedback since september. There seems to be a merge conflict now. Can you rebase the master, so we can test it again?

**Comment by Mamaduka** [comment]
Hi, @truchot

Sorry, I think this PR fell into the cracks. The original issue was resolved via #21944, and the "No results" state for Inserter has been improved since then

I'm going to close this PR.

Thanks for your contributions!

## Screenshot

CleanShot 2022-09-26 at 13 12 13

**Pull request closed without merging**